src/dom0.c

```diff
@@ -44,7 +48,16 @@ int main(void)
 		goto exit_err;
 	}
 
-	/* It's required to init xenlib struct xen_domain_cfg->image_info with pointer
+	/*
+	 * Start initialization routine for the pv_domain to parse
+	 * vbd and vif configuration strings and fillup backed parameter
+	 * data.
+	 */
+	if (IS_ENABLED(CONFIG_DOM_CFG_LINUX_PV_DOMAIN)) {
+		fill_pv_params();
```

Comment:
I'd prefer if you add `.init() `callback in `struct dom0_domain_cfg, `
which accepts `struct dom0_domain_cfg *dom0_cfg` as parameter

Reply:
This will affect too many entities, such as zephyr-xenlib, xrun tests and so on. I will leave todo here.

Reply:
I think I've found solution. please ignore previous message

Reply:
fixed